easy_maps/templatetags/map_tags.py

```diff
@@ -22,6 +22,7 @@ class MapTag(InclusionTag):
     options = Options(
         Argument('lat', resolve=True, required=True),
         Argument('lng', resolve=True, required=True),
+        Argument('setMarker', required=False, default=False),
```

Comment:
standardul in python e underline si asa am folosit pana acuma. as pune `default=True` totusi pentru backwards compatibility. also l-as pune la sfarsit dupa zoom, tot pentru backwards compatibility, pentru ca in momentul asta formularul de pe edit.html nu functioneaza.